tools/minibrowser/build.gradle

```diff
@@ -1,8 +1,20 @@
+buildscript {
+    repositories {
+        google()
+    }
+
+    dependencies {
+        classpath 'androidx.navigation:navigation-safe-args-gradle-plugin:2.5.2'
+    }
+}
+
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
```

Comment:
Maybe you can move here the line `id 'org.jetbrains.kotlin.android' version '1.7.20' apply false` from the root build.gradle file as we don't need to load the kotlin plugin anywhere else (that would be more coherent). You will have to remove the `apply false` from this line.

You can also add the `androidx.navigation.safeargs.kotlin`  plugin here:
```
    id 'androidx.navigation.safeargs.kotlin' version '2.5.2'
```

Reply:
I'll try it